app/assets/stylesheets/styles.scss

```diff
@@ -40,25 +46,4 @@ li {
 .side-panel-image {
   text-align: center;
   margin: 1.5em;
-}
-
-.favorite-button, .unfavorite-button {
```

Comment:
And all of the below needs to go back in? Does it matter what lines I put it on? 

> On Feb 2, 2015, at 1:52 AM, Ryan Mitchell <notifications@github.com> wrote:
> 
> > @@ -40,25 +46,4 @@ li {
> >  .side-panel-image {
> >    text-align: center;
> >    margin: 1.5em;
> > -}
> > -
> > -.favorite-button, .unfavorite-button {